### DISCUSSION TITLE
Welcome to Nexth Discussions!

### DISCUSSION THREAD

**Discussion by wslyvh** [wslyvh/nexth#9] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us how you're using Nexth or what you'd like to see included for it to be more useful?  of yourself and tell us about what you do with this community.

**Comment by DanielaS-Tochi** [comment]
Hi! I'm Daniela from Argentina. I've started learning NextJs and I'd like to be a Web3 developer. I'll make use of your repository to join Next and Ethereum, is that correct? 
PD. Sorry for my English 😁
